nesta/packages/crunchbase/crunchbase_collect.py

```diff
@@ -135,17 +138,12 @@ def process_orgs(orgs, existing_orgs, cat_groups, org_descriptions):
     # fix uuid column names
     orgs = rename_uuid_columns(orgs)
 
-    # change NaNs to None
-    orgs = orgs.where(orgs.notnull(), None)
-    org_descriptions = org_descriptions.where(org_descriptions.notnull(), None)
-
     # lookup country name and add as a column
     orgs['country'] = orgs['country_code'].apply(country_iso_code_to_name)
-    orgs = orgs.drop('country_code', axis=1)  # now redundant with country_alpha_3 appended
-
+    
     orgs['long_description'] = None
     org_cats = []
-    cat_groups = cat_groups.set_index(['category_name'])
+    cat_groups = cat_groups.set_index(['name'])
```

Comment:
example of variable renaming between versions 3 and 4 - there are many examples of this that you will see throughout